README.md

```diff
@@ -39,3 +39,11 @@ direnv allow
 ```
 
 Now, you don't need to type `nix develop` anymore.  Any time you change into the project directory, direnv will automatically load the development shell, and it will automatically unload the development shell when you navigate out of the project directory.  An additional benefit is that it will integrate with your existing shell of choice, e.g. bash, zsh, etc.
+
+### htmx-go branch
+
+make sure .env file is set up
```

Comment:
We should add the details here for new users.  In other words, explain that a `.env` file needs to be created at the root of the project with the variables set:
```env
BTCUSER=my_bitcoind_username
PASSWORD=my_bitcoind_password
```